salt/modules/zypperpkg.py

```diff
@@ -2133,6 +2133,75 @@ def purge(name=None, pkgs=None, root=None, **kwargs):  # pylint: disable=unused-
     return _uninstall(name=name, pkgs=pkgs, root=root)
 
 
+def removeptf(
```

Comment:
Why do we need an entirely new function? Why not add a new arg `ptf` to the current `remove` function and add logic to use the correct zypper command if that arg is set?

Reply:
we are trying out a second option currently